### ISSUE TITLE
	AX_TEXT_01 Accessibility Audit test should probably ignore role=presentation elements

### ISSUE THREAD

**Issue by alice** [GoogleChrome/accessibility-developer-tools#97]
Reported by hickerne@google.com on Chromium issue tracker: https://code.google.com/p/chromium/issues/detail?id=398482

UserAgent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/36.0.1985.125 Safari/537.36

Steps to reproduce the problem:
1. Run the Accessibility audit on a page containing an <input> element with an ARIA role of presentation
2. The AX_TEXT_01 test fails.

What is the expected behavior?

What went wrong?
I think that the test should have passed.  According to http://www.w3.org/TR/wai-aria/roles#presentation, elements with this role should not be mapped to the accessibility API.  I might be misreading the spec, but I think this means that elements containing that role should not need a label.

Did this work before? No
